warehouse/query-core/src/main/java/datawave/query/jexl/functions/GroupingRequiredFilterFunctions.java

```diff
@@ -133,6 +135,17 @@ private static boolean manageGroupsForMatchesInGroupRemainingArgs(Object fieldVa
         }
     }
     
+    // move all these kinds of methods into a central utility once we refactor this
+    private static String getSubgroup(String fieldName) {
+        String[] splits = StringUtils.split(fieldName, '.');
```

Comment:
So yes, this will do the job.  However a little more efficient might be:
```
int index = fieldName.lastIndexOf('.');
if (index > 0) {
  return fieldName.substring(index+1);
}
return null;
```

Reply:
+1